src/main/java/javaserver/App.java

```diff
@@ -52,22 +54,24 @@ public static Authenticator makeCobSpecAuthenticator() {
         return authenticator;
     }
 
-    public static Router makeCobSpecRouter(String directory) {
+    private static Router makeCobSpecRouter(String directory) {
         Router router = new Router(directory);
-        router.addRoute(new Route("/form", "GET", new FileHandler(directory + "/form")));
-        router.addRoute(new Route("/form", "POST", new PostPutHandler(directory + "/form")));
-        router.addRoute(new Route("/form", "PUT", new PostPutHandler(directory + "/form")));
-        router.addRoute(new Route("/form", "DELETE", new DeleteHandler(directory + "/form")));
+        FileUtility fileUtility = new FileUtility();
+
+        router.addRoute(new Route("/form", "GET", new FormHandler(directory + "/form", fileUtility)));
```

Comment:
Would an enum be overkill for this? I use these strings a few times project wide.